apps/test-viewer/src/components/ViewerOptions.tsx

```diff
@@ -100,6 +100,7 @@ function ToggleTreeNodesSelectionButton() {
 }
 
 function SelectionScopeSelectorButton() {
+  // eslint-disable-next-line deprecation/deprecation
```

Comment:
This needs to be adjusted so it doesn't require usage of deprecated APIs. I can do that later this week.

Reply:
looks like the latest viewer RC packages don't have the latest changes from 4.x. will have to wait until they're merged-in and released.

Reply:
Will tackle this is as a separate pr, I need to update the viewer to pull in the changes you had made for this and cut a new release. Dont want to block this pr for that